crates/api/src/wasmtime_unit.rs

```diff
@@ -20,7 +20,15 @@ pub(crate) struct Wasmtime {
 
 impl Wasmtime {
     pub fn new(mounts: Vec<DirectoryMount>) -> Self {
-        let store = Store::default();
+        let mut config = wasmtime::Config::new();
+        if let Err(e) = config
+            .static_memory_maximum_size(536_870_912)
+            .strategy(Strategy::Lightbeam)
```

Comment:
Lightbeam is an unstable backend for Wasmtime, plus it's a single-pass optimising JIT. Is there any reason we would prefer that to Cranelift? Also, lightbeam requires nightly AFAIK.

Reply:
Actually, now that I think of it, it could be worth having two WASI plugins: one using Cranelift and one using Lightbeam. Then the requestor would have the freedom to choose which one they want, etc.

Reply:
performance under graphene is very poor. I was looking for backed with less `clock_gettime`, `map`, `unmap` syscalls.